src/components/FooterSocial.astro

```diff
@@ -0,0 +1,41 @@
+---
+import { Icon } from 'astro-icon';
+
+const { href, link, title, icon, icon_dark } = Astro.props;
+---
+
+<a
+    href={href || link}
+    rel={href && 'me'}
+    target={href && '_blank'}
+    title={title}
+    data-bs-toggle="tooltip"
+    data-bs-placement="bottom"
+>
+    <Icon name={icon} class={icon_dark && 'hide-dark'} />
+    {icon_dark && <Icon name={icon_dark} color="white" class="hide-light" />}
+</a>
+
+<style lang="scss">
+    @import 'bootstrap/scss/bootstrap.scss';
+    @import 'bootstrap-print-css/css/bootstrap-print.min';
+    @import '../styles/_variables.scss';
+    a {
+        margin-left: 6px;
```

Comment:
can we try to use `rem` for margins? Leads to more consistency.

Reply:
Done 👍🏻